src/docs/user_ai.md

```diff
@@ -279,13 +283,34 @@ The Theia IDE provides a global chat interface where users can interact with all
 
 <img src="../../general-chat.png" alt="General AI Chat in the Theia IDE" style="max-width: 525px">
 
-Some agents produce special results, such as buttons (shown in the screenshot above) or code that can be directly inserted. You can augment your requests in the chat with context by using variables. For example, to refer to the currently selected text, use `#selectedText` in your request. Pressing '#' in the chat will show a list of available variables.
+Some agents produce special results, such as buttons (shown in the screenshot above) or code that can be directly inserted. 
+
+### Agent Pinning
+The *Agent Pinning* feature, reduces the need for repeated agent references.  
+  
+- When you mention an agent in a prompt and no agent is pinned, the mentioned agent is automatically pinned.  
+- If an agent is already pinned, mentioning a different agent will **not** change the pinned agent. Instead, the newly mentioned agent will be used **only** for that specific prompt.  
+- You can manually unpin an agent through the chat interface if needed.  
+- A new chat session can now be started directly with a pinned agent.
```

Comment:
```suggestion
```

There is no command anymore to start a new chat with a pinned agent. Instead the @ symbol triggers the agent selection and then the rules above just apply. So creating a new chat with a pinned agent is essentially now starting a new chat and selecting an agent in the first message.